utils/openshift_resource.py

```diff
@@ -406,16 +406,24 @@ def initialize_resource_type(self, cluster, namespace, resource_type):
 
     def add_desired(self, cluster, namespace, resource_type, name, value):
         with self._lock:
-            desired = \
-                self._clusters[cluster][namespace][resource_type]['desired']
+            try:
+                desired = \
+                    (self._clusters[cluster][namespace][resource_type]
+                        ['desired'])
+            except KeyError:
+                return False
```

Comment:
Even if we need it, we should probably return `None`.

Reply:
I based it on other `KeyError` which used `False` but will switch to `None`

Reply:
All I care is what the function is giving back to the caller so it does not break.

Reply:
But I don't think that the return from `add_desired()` is ever used, so there's no reason to return a boolean.

Reply:
:+1: